contrib/opencensus-ext-requests/setup.py

```diff
@@ -12,13 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+
 from setuptools import find_packages, setup
 
-from version import __version__
+BASE_DIR = os.path.dirname(__file__)
+VERSION_FILENAME = os.path.join(
+    BASE_DIR, "opencensus", "ext", "requests", "common", "version.py"
+)
+PACKAGE_INFO = {}
+with open(VERSION_FILENAME) as f:
+    exec(f.read(), PACKAGE_INFO)
```

Comment:
Why do the other packages in this PR import the `version` module directly but this one reads the file and `exec`s it? I think we do the latter in OTel, but not sure of the difference between the two.

Reply:
Great question. `opencensus-ext-request` imports the [trace](https://github.com/census-instrumentation/opencensus-python/blob/master/contrib/opencensus-ext-requests/opencensus/ext/requests/__init__.py#L15) module in one of it's `__init__.py`, in which it imports [requests](https://github.com/census-instrumentation/opencensus-python/blob/master/contrib/opencensus-ext-requests/opencensus/ext/requests/trace.py#L17). So if we were to import the `version` module directly in `setup.py`, the imports will attempt to resolve first before `requests` can be installed which will result in `requests` not being able to be found. For consistency, we can make all of them read from the file. Not a strong opinion.

Reply:
That would be great if they can all jsut read the file, thanks for explaining